python-sdk/dev/docker-compose.yaml

```diff
@@ -21,6 +21,8 @@ x-airflow-common:
     ASTRONOMER_ENVIRONMENT: local
     AIRFLOW__LINEAGE__BACKEND: openlineage.lineage_backend.OpenLineageBackend
```

Comment:
You don't need this on Airflow 2.3+

Reply:
astro-sdk works on `airflow >2.0`. So it would be good to keep it